velox/docs/functions.rst

```diff
@@ -61,61 +61,62 @@ for :doc:`all <functions/coverage>` and :doc:`most used
     ======================================  ======================================  ======================================  ==  ======================================
     Scalar Functions                                                                                                            Aggregate Functions
     ======================================================================================================================  ==  ======================================
-    :func:`abs`                             :func:`empty_approx_set`                :func:`pow`                                 :func:`approx_distinct`
```

Comment:
The PR introducing a new function should only add documentation for that function. It should not update coverage docs. These should be updated in a follow-up PR.